quest/src/core/errors.cpp

```diff
@@ -48,7 +48,10 @@ void raiseInternalError(string errorMsg) {
 void error_functionNotImplemented(const char* caller) {
 
     string name = caller;
-    raiseInternalError("The function '" + name + "' has not yet been implemented.");
+    
+    // DEBUG
```

Comment:
oops this whole function is no longer called :tada: Nothing to edit here though, I'll delete it on a subsequent commit

Reply:
Noted!